acquia_lift.admin.wizard.inc

```diff
@@ -702,23 +710,22 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
       }
     }
   }
-  if ($has_tests) {
-    $status = personalize_agent_get_status($agent_data->machine_name);
-    $editable = $status === PERSONALIZE_STATUS_NOT_STARTED || $status === PERSONALIZE_STATUS_PAUSED;
-
-    if (!$editable) {
-      if ($status === PERSONALIZE_STATUS_COMPLETED) {
-        drupal_set_message(t('Test options can no longer be edited.'), 'warning');
-      }
-      else {
-        drupal_set_message(t('Test options cannot be edited while this personalization is running.'), 'warning');
-      }
-    }
+  // If there is only one audience then there is no explicit targeting so
+  // check for auto-targeting options.
+  $context_element = FALSE;
+  if ($count_audiences < 2) {
+    $visitor_context = isset($agent_data->data['visitor_context']) ? $agent_data->data['visitor_context'] : array();
+    $context_element = personalize_admin_build_visitor_context_select($visitor_context, TRUE, $agent_data);
+  }
+  $has_advanced = $has_tests || $context_element !== FALSE;
```

Comment:
I know what's going on here... the check on $has_tests is whether to show the test options (decision_style, etc.) and the check for $context_element is actually for the visitor_contexts.  Both of those things go into the "advanced" section so they both get utilized here.

Of course this changes after Fridays' review comments anyway now that the cache decisions got moved in there.